sdk/servicebus/service-bus/test/public/invalidParameters.spec.ts

```diff
@@ -133,6 +133,22 @@ describe("invalid parameters", () => {
       );
     });
 
+    [false, 0, -0, "", NaN].forEach((falsyValue) => {
```

Comment:
IIUC, false and "" are not valid Long values, right? that is why you're casting to any in line 140. So I wonder why do we care about those values if the type doesn't allow them?

Reply:
TypeScript would give an error for these types.  However, JavaScript customers won't, and they currently don't get the expected type error.  I listed all falsy values here but I agree that it's unlikely customers would pass in boolean value or empty strings.

Reply:
even `0` is not a valid `Long` value.  This is why casting is needed to test the runtime error.